src/app/wallets/update-pending-invoices.ts

```diff
@@ -151,22 +133,26 @@ const updatePendingInvoice = async ({
       return false
     }
     if (invoiceToUpdate instanceof Error) return invoiceToUpdate
-    if (invoiceToUpdate.paid) {
+
+    if (walletInvoice.paid) {
       pendingInvoiceLogger.info("invoice has already been processed")
       return true
     }
 
     const displayCurrencyPerSat = await getCurrentPrice()
     if (displayCurrencyPerSat instanceof Error) return displayCurrencyPerSat
 
+    const invoiceSettled = await lndService.settleInvoice({ pubkey, secret })
+    if (invoiceSettled instanceof Error) return invoiceSettled
+
+    const invoicePaid = await walletInvoicesRepo.markAsPaid(paymentHash)
+    if (invoicePaid instanceof Error) return invoicePaid
+
     // TODO: this should be a in a mongodb transaction session with the ledger transaction below
```

Comment:
Just checking if this comment is still relevant here

Reply:
yes still relevant